pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -37,5 +37,9 @@ python3Packages.buildPythonApplication rec {
     homepage = "https://zeronet.io/";
     license = licenses.gpl2;
     maintainers = with maintainers; [ fgaz ];
+    knownVulnerabilities = [ ''
+      Unmaintained. Probable XSS/code injection vulnerability.
+      Switching to the maintained zeronet-conservancy package is recommended
+    '' ];
```

Comment:
If you don't use it yourself, why not drop this package. Interested parties can use the original package by overriding the fork.

Reply:
but then what should we do with the zeronet module?

* set `package` to `zeronet-conservancy`. feels a bit sneaky, though we'd mention it in the changelog so i guess it could be fine?
* leave it as is, so that users would get an error about a missing package instead of an error about insecurity
* rename the module to `zeronet-conservancy`

Reply:
Good point. I wonder if it's okay to default the `service.zeronet.package` to `null`, so that user has to supply a fork of their choice. The current option of using `zeronet.meta.knownVulnerabilities` is fine too, I guess.